packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -1052,6 +1052,259 @@ describe('RichTextEditor', () => {
+                        url: 'https://nimble.ni.dev/'
+                    },
+                    {
+                        name: 'Hyperlink(https) in nimble-anchor',
+                        input: '<nimble-anchor href="https://nimble.ni.dev/"><a href="https://nimble.ni.dev/">Nimble</a></nimble-anchor>',
+                        url: 'https://nimble.ni.dev/'
+                    },
+                    {
+                        name: 'Absolute link(HttPs) in nimble-anchor as mixed case',
+                        input: '<nimble-anchor href="HttPs://nimble.ni.dev/"><a href="HttPs://nimble.ni.dev/">HttPs://nimble.ni.dev/</a></nimble-anchor>',
+                        url: 'HttPs://nimble.ni.dev/'
+                    },
+                    {
+                        name: 'Hyperlink(HttPs) in nimble-anchor as mixed case',
+                        input: '<nimble-anchor href="HttPs://nimble.ni.dev/"><a href="HttPs://nimble.ni.dev/">Nimble</a></nimble-anchor>',
+                        url: 'HttPs://nimble.ni.dev/'
+                    },
+                    {
+                        name: 'Absolute link(http) in nimble-anchor as upper case',
+                        input: '<nimble-anchor href="http://nimble.ni.dev/"><a href="http://nimble.ni.dev/">http://nimble.ni.dev/</a></nimble-anchor>',
+                        url: 'http://nimble.ni.dev/'
+                    },
+                    {
+                        name: 'Hyperlink(HTTP) in nimble-anchor as upper case',
+                        input: '<nimble-anchor href="HTTP://nimble.ni.dev/"><a href="HTTP://nimble.ni.dev/">Nimble</a></nimble-anchor>',
+                        url: 'HTTP://nimble.ni.dev/'
+                    }
+                ] as const;
+
+                parameterizeNamedList(
+                    differentValidLinks,
+                    (spec, name, value) => {
+                        spec(
+                            `${name} renders as absolute link(href and text content should be same) in editor`,
+                            () => {
+                                pageObject.pasteHTMLToEditor(value.input);
+
+                                expect(pageObject.getEditorTagNames()).toEqual([
+                                    'P',
+                                    'A'
+                                ]);
+                                expect(
+                                    pageObject.getEditorLeafContents()
+                                ).toEqual([value.url]);
+                                expect(
+                                    pageObject.getEditorLastChildAttribute(
+                                        'href'
+                                    )
+                                ).toBe(value.url);
+                            }
+                        );
+                    }
+                );
+            });
+
+            describe('pasting various not supported links should render as plain text', () => {
```

Comment:
Could you add one case that pastes a plain text URL and validates that renders as plain text (unless you already cover that somewhere and I missed it)?

Reply:
Added a test case.